src/Microsoft.Sbom.Targets/SbomInputValidator.cs

```diff
@@ -41,18 +41,18 @@ public bool ValidateAndSanitizeRequiredParams()
             }
         }
 
-        this.PackageSupplier = Remove_Spaces_Tabs_Newlines(this.PackageSupplier);
-        this.PackageName = Remove_Spaces_Tabs_Newlines(this.PackageName);
-        this.PackageVersion = Remove_Spaces_Tabs_Newlines(this.PackageVersion);
+        this.PackageSupplier = Remove_Tabs_Newlines(this.PackageSupplier);
+        this.PackageName = Remove_Tabs_Newlines(this.PackageName);
+        this.PackageVersion = Remove_Tabs_Newlines(this.PackageVersion);
```

Comment:
PackageSupplier and PackageName should definitely allow inner spaces, but I'm less convinced about PackageVersion. Based on the docs at https://learn.microsoft.com/en-us/nuget/concepts/package-versioning and the parsing tests at https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.Tests/NuGet.Versioning.Test/NuGetVersionTest.cs#L90-L166 (this library is recommended from https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort#where-nugetversion-diverges-from-semantic-versioning), I would expect that the PackageVersion string would _not_ allow inner spaces